tests/test_FLEbasis2D.py

```diff
@@ -0,0 +1,330 @@
+import os
+
+import numpy as np
+import pytest
+
+from aspire.basis import FBBasis2D, FLEBasis2D
+from aspire.image import Image
+from aspire.nufft import backend_available
+from aspire.numeric import fft
+from aspire.source import Simulation
+from aspire.utils import utest_tolerance
+from aspire.volume import Volume
+
+from ._basis_util import UniversalBasisMixin
+
+DATA_DIR = os.path.join(os.path.dirname(__file__), "saved_test_data")
+
+
+def show_fle_params(basis):
+    return f"{basis.nres}-{basis.epsilon}"
+
+
+def gpu_ci_skip():
+    pytest.skip("1e-7 precision for FLEBasis2D.evaluate()")
+
+
+def odd_resolution_skip():
+    pytest.skip("FB matching for odd resolution.")
+
+
+fle_params = [
+    (32, 1e-4),
+    (32, 1e-7),
+    (32, 1e-10),
+    (32, 1e-14),
+    (33, 1e-4),
+    (33, 1e-7),
+    (33, 1e-10),
+    (33, 1e-14),
+]
+
+test_bases = [
```

Comment:
This would have to interact with the `UniversalBasisMixin` methods, which currently take a `basis` argument but would have to take something like a `basis_fixture` argument, recognized across the session. I think this would be best done while also creating fixtures for the other bases (or creating a generalized fixture in `_basis_util`. Issue?

Reply:
Sure, I think that is a direction we will go in the future. Thanks

Reply:
Created #855